package.json

```diff
@@ -37,7 +37,8 @@
     "prettier-fix": "prettier --write .",
     "prettier-check": "prettier --check .",
     "prepare": "npm run build",
-    "publish-all": "npm publish --workspaces --access public && npm publish --access public"
+    "publish-all": "npm publish --workspaces --access public && npm publish --access public",
+    "test:e2e": "npm run test:e2e --workspace=client"
```

Comment:
Please move this one up the list with the other two test scripts.

Reply:
Okay, done in a2c97a0.